tests/assembly/manual-eq-efficient.rs

```diff
@@ -0,0 +1,25 @@
+// Regression test for #106269
+//@ revisions: WIN LIN
+//@ [WIN] only-windows
+//@ [LIN] only-linux
```

Comment:
Why do we need a bunch of revisions and such? e.g., tests/assembly/x86_64-cmp.rs doesn't have windows/linux revisions.

Reply:
Mostly there so I could check specifically for the registers used on those platforms. I can switch it to just using Regex though